### ISSUE TITLE
Easy Map: absence records mapped as presence records

### ISSUE THREAD

**Issue by CEJohnston** [nbnuk/nbnatlas-issues#280]
Bug raised on the Forum:  [https://forums.nbn.org.uk/viewtopic.php?id=6928](url)

Compare the EasyMap for Wakame [https://easymap.nbnatlas.org/Image?tvk=NHMSYS0021058672&w=600](url) 

with the Atlas map [https://records.nbnatlas.org/occurrences/search?q=lsid:NHMSYS0021058672#tab_mapView](url)

The EasyMap shows more records.

Selecting an area on the Atlas map where the EasyMap shows records but the Atlas map is blank, e.g. Tyneside, finds records such as [https://records.nbnatlas.org/occurrences/bfda7bfe-1e1b-44c0-8dbe-8fc4c68edf61](url)
This is an absence record.

My conclusion is that the EasyMap does not distinguish between presence and absence records and therefore gives a false impression of distribution.

**Comment by sophiathirza** [comment]
I have forked the original EasyMap_Shim repository into nbnUK. The additional query string parameter (fq=-occurrence_status=absent) has been added to the web service calls. 

This now just needs to be made live. 

I had to change all readall() method calls on httpresponse objects to read() in order for the code to work locally: https://www.bountysource.com/issues/30244474-httpresponse-object-has-no-attribute-readall. We would check that this is okay on production.

I also was not able to test the code locally using ssl options, but I think that is okay.
